go/registry/api/runtime.go

```diff
@@ -115,9 +112,6 @@ func (e *ExecutorParameters) ValidateBasic() error {
 
 // TxnSchedulerParameters are parameters for the runtime transaction scheduler.
 type TxnSchedulerParameters struct {
```

Comment:
`fixgenesis` is not needed for this removal right, as the old field will just get skipped when deserializing the json, right?

Reply:
That's right.